### PR TITLE
Fixed Safe connector for contract writes

### PR DESCRIPTION
**Pull request by dasanra**
## Description

Fixed issue https://github.com/wevm/viem/issues/579 bumping `@safe-global/safe-apps-provider@0.18.1`.

Related issue https://github.com/safe-global/safe-apps-sdk/issues/480

## Additional Information

- [x] I read the [contributing docs](/wagmi-dev/wagmi/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

### CODE REVIEW

File: .changeset/nervous-tigers-smash.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@wagmi/connectors": patch
```
Comment (tmm) [inline .changeset/nervous-tigers-smash.md:2]: You also want to add `wagmi` and `@wagmi/core` to this list so they get the update downstream too!

File: .changeset/nervous-tigers-smash.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+Fixed issue https://github.com/wevm/viem/issues/579 bumping `@safe-global/safe-apps-provider@0.18.1`
```
Comment (tmm) [inline .changeset/nervous-tigers-smash.md:5]:
```suggestion
Fixed issue with [Safe SDK](https://github.com/wevm/viem/issues/579) by bumping `@safe-global/safe-apps-provider@0.18.1`
```

**Review by tmm — CHANGES_REQUESTED** [review]
Small tweaks, then will be good to go.

**Comment by tura11541** [comment]
Do not deploy

tura

**Comment by tura11541** [comment]
Fix it so everything operates accordingly..

tura

**Comment by tura11541** [comment]
And operate legally..
Please no sneak attack codes
No more ...
And take note ...
After today no more disruptive codes ...
Even If tomorrow's request.. no more negative codes ..
Put that in writing, put it in a quote. Pass it on along with yelver
assistant helpers with github. No more negative codes, no more disruptive
code. To the process. Please no more negative codes.

tura

File: .changeset/nervous-tigers-smash.md (hunk 1 above)
Comment (dasanra) [reply .changeset/nervous-tigers-smash.md:2]: Done 👍

**Pull request merged**